### DISCUSSION TITLE
Debian bullseye testing

### DISCUSSION THREAD

**Discussion by git-aniket** [OpenBangla/OpenBangla-Keyboard#196] [Q&A] [answered]
Is there a build available for Debian bullseye(testing), since it soon goes to freeze?. While building from source in bullseye, seems like the qt5-default package required for the build is not available in the default repositories.

Adding the repositories for OBK doesn't work either, giving a "Malformed entry in /etc/apt/sources.list.d/openbangla.list" error.

Is there an alternative way to build it for Debian bullseye?

**Comment by git-aniket** [comment, marked as answer]
Ok, Figured it out finally.
In order to use OBK for Bullseye, one has to compile it from its source. 
Just that the Bullseye seems to have rearranged some packages in its repo's, and has no package candidate called 'qt5-default'.
Instead one needs to install the qt5base-dev instead. 

_sudo apt-get install build-essential rustc cargo cmake libibus-1.0-dev qtbase5-dev libzstd-dev_
Then compile and restart the PC.
https://packages.debian.org/bullseye/qtbase5-dev 
Once that was done it worked like charm for me!!!

**Reply by mominul** [reply]
That's great! But it's quite worrying that they have changed the package name. :worried: I think we'll need to update the Readme and package description.
